harvester_e2e_tests/fixtures/images.py

```diff
@@ -14,6 +14,7 @@
 @pytest.fixture(scope="session")
 def image_opensuse(request, api_client):
     image_server = request.config.getoption("--image-cache-url")
+    checksum = request.config.getoption("--opensuse-checksum")
```

Comment:
You can change the config to just be `""` and it works fine. It doesn't require you to add a checksum, and the tests will still run fine without one. I left a default in since that's what we're running in our CI builds.

Reply:
I removed the default to not have a checksum and all of the tests pass.

Reply:
I added in the default, and changed the name to be more specific.